src/graphorm/index.ts

```diff
@@ -222,14 +223,35 @@ const obj = new GraphORM({
     requiredColumns: ['createdAt', 'userId', 'role'],
     fields: {
       permissions: {
-        transform: (_, ctx: Context, member: SourceMember) => {
+        select: (ctx: Context, alias: string, qb: QueryBuilder): string => {
+          const query = qb
+            .select('"memberPostingRank"')
+            .from(Source, 'ps')
```

Comment:
Not sure what ps stand for in the context

Reply:
`posting squad`, Was not sure if `source` will collide with anything else. I'll make it more verbose.